### PR TITLE
fix:(tabs) add call for rechecking tab scroll distance

### LINKED ISSUE #12889: Tabs partially hidden by arrows
#### Bug, feature request, or proposal:
https://stackblitz.com/edit/angular-gkiyz4?file=app/tab-group-basic-example.html

The last tab is selected by default, with a small screen the tab is partially hidden by the arrow. I have to click on the right arrow to display the tab.

#### What is the expected behavior?
Display entirely the tab.

#### What is the current behavior?
The tab is partially hidden by the arrow

#### What are the steps to reproduce?
https://stackblitz.com/edit/angular-gkiyz4?file=app/tab-group-basic-example.html

#### Which versions of Angular, Material, OS, TypeScript, browsers are affected?
```json
"dependencies": {
    "@angular/animations": "^6.1.2",
    "@angular/cdk": "^6.4.3",
    "@angular/common": "^6.1.2",
    "@angular/compiler": "^6.1.4",
    "@angular/core": "^6.1.4",
    "@angular/forms": "^6.1.4",
    "@angular/http": "^6.1.4",
    "@angular/material": "^6.4.6",
    "@angular/platform-browser": "^6.1.4",
    "@angular/platform-browser-dynamic": "^6.1.4",
    "@angular/router": "^6.1.4",
    "material-design-icons": "^3.0.1",
  }
```
Chrome and FF latest.

#### Is there anything else we should know?
Nope

### PR DESCRIPTION
**Pull request by lau32**
Add call to scroll the tabs header after initialising header pagination, to accurately calculate the scroll distance.

Fixes #12889

### CODE REVIEW

**Comment by lau32** [comment]
@crisbeto, would you see any other test cases?

File: src/lib/tabs/tab-header.spec.ts [hunk 1]
```diff
@@ -382,6 +382,23 @@ describe('MatTabHeader', () => {
     });
 
   });
+
+    it('scroll to last tab', () => {
```
Comment (crisbeto) [inline src/lib/tabs/tab-header.spec.ts]: I'm not sure that this tests the issue that was reported in #12889. It looks like it only tests that `_scrollToLabel` works.

**Comment by Carniatto** [comment]
any updates on this PR?

**Comment by lau32** [comment]
@Carniatto I'll close this one and create another one.

**Pull request closed without merging**